elasticsearch/sgconfig/sg_rolesmapping.yml

```diff
@@ -0,0 +1,15 @@
+sg_role_kibana:
+  users:
+    - 'CN=system.logging.kibana,OU=OpenShift,O=Logging,L=Test,C=DE'
```

Comment:
`CN=system.logging.kibana,OU=OpenShift,O=Logging` works for me - afaik, no need for L or C.  If for some reason it is an SG requirement, then I suggest L=Raleigh,c=US